src/cli/command.rs

```diff
@@ -62,7 +62,7 @@ pub enum Command {
     Gen {
         /// Path of the source files separated by spaces if more than one
         #[arg(required = true)]
-        file_paths: Vec<String>,
+        paths: Vec<String>,
```

Comment:
update docs on the website repo

Reply:
will raise pr for doc change once this pr get's approved.